core-bundle/src/Doctrine/Backup/Dumper.php

```diff
@@ -117,8 +112,16 @@ private function formatValueForDump($value, Column $column, Connection $connecti
             return 'NULL';
         }
 
-        if ($column->getType() instanceof BinaryType) {
-            return sprintf('UNHEX(%s)', $connection->quote($value));
+        // non-ASCII values
+        if (
+            \is_string($value)
```

Comment:
Can we have a test for that in `DumperTest` maybe? ☺️

Reply:
Done in 0733330e9b7bb40bacfdc1ef19815fe785673de8